### ISSUE TITLE
The 'crfrnn_layer' layer is used in the last layer of the network. When the accuracy of the network reaches about 0.2, the loss function returns nan

### ISSUE THREAD

**Issue by drdlfy** [sadeepj/crfasrnn_keras#14]
Thank you very much for your code. But I  use the 'crfrnn_layer' layer in the last layer of my network, when training my network, when the network accuracy rate of 0.2 or so, the loss function will return to nan, my data with RGB pictures or TIFF will If this happens, please help solve this problem. Looking forward to your reply, thank you very much.
The following is the training process:
81/2560 [...........] - ETA: 948s - loss: 6.5810 - binary_crossentropy: 0.5113 - accuracy: 0.1502
82/2560 [...........] - ETA: 944s - loss: 6.4899 - binary_crossentropy: 0.5118 - accuracy: 0.1719
83/2560 [...........] - ETA: 941s - loss: 6.4444 - binary_crossentropy: 0.5107 - accuracy: 0.1739
84/2560 [...........] - ETA: 940s - loss: 6.3637 - binary_crossentropy: 0.5116 - accuracy: 0.1921
85/2560 [...........] - ETA: 937s - loss: nan - binary_crossentropy: 0.5131 - accuracy: 0.1473
86/2560 [>..........] - ETA: 935s - loss: nan - binary_crossentropy: 0.5137 - accuracy: 0.1514

**Comment by sadeepj** [comment]
@drdlfy I'm unable to be of direct help without seeing your dataset, network hyper-parameters etc. However, I can say that this usually happens when the learning rate is too high. If you're training something from scratch, I would suggest training only the FCN part first and then fine-tuning with CRF-RNN. Section 6 and 7 of our paper http://www.robots.ox.ac.uk/~szheng/papers/CRFasRNN.pdf contains some useful details about training CRF-RNN networks.

**Comment by drdlfy** [comment]
@sadeepj  Yes, thank you very much for your help！
